### PR TITLE
Appsee iOS Kit - Initial commit

### PR DESCRIPTION
**Pull request by roital**

### CODE REVIEW

File: mParticle-Appsee.podspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                       This is the Appsee integration for mParticle.
+                       DESC
+
+    s.homepage         = "https://www.mparticle.com"
+    s.license          = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.author           = { "mParticle" => "support@mparticle.com" }
+    s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-appsee.git", :tag => s.version.to_s }
+    s.social_media_url = "https://twitter.com/mparticles"
+
+    s.ios.deployment_target = "6.0"
```
Comment (samdozor) [inline mParticle-Appsee.podspec]: Can you change this to 8.0

File: mParticle-Appsee.podspec [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+    s.name             = "mParticle-Appsee"
+    s.version          = "2.4.1"
```
Comment (samdozor) [inline mParticle-Appsee.podspec]: This version will ultimately change to make our current SDK version, 7.5.x

File: mParticle-Appsee.podspec [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    s.homepage         = "https://www.mparticle.com"
+    s.license          = { :type => 'Apache 2.0', :file => 'LICENSE' }
+    s.author           = { "mParticle" => "support@mparticle.com" }
+    s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-appsee.git", :tag => s.version.to_s }
+    s.social_media_url = "https://twitter.com/mparticles"
+
+    s.ios.deployment_target = "6.0"
+    s.ios.source_files      = 'mParticle-Appsee/*.{h,m,mm}'
+    s.ios.dependency 'mParticle-Apple-SDK/mParticle'
```
Comment (samdozor) [inline mParticle-Appsee.podspec]: should be `s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 7.5.0'`

**Review by samdozor — CHANGES_REQUESTED** [review]
Looks pretty good! Left a few comments inline.

Also - I noticed that you have an xcodeproj but do not support carthage - are you able to add support for that?

File: mParticle-Appsee.podspec (hunk 2 above)
Comment (oriori5) [reply mParticle-Appsee.podspec]: In such case, how will we be able to version our kit?

**Review by samdozor — APPROVED** [review]
This looks great. I've moved it to a dedicate repository which you can see here: 

https://github.com/mparticle-integrations/mparticle-apple-integration-appsee

**Pull request closed without merging**